### PR TITLE
Rework deploying a configuration

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Reworked "deploying a configuration"

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

### CODE REVIEW

**Comment by cg-tw** [comment]
https://jenkins.centreon.com/job/centreon-documentation/view/change-requests/job/PR-805/Centreon_20documentation_20preview/

File: en/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 1]
```diff
@@ -3,39 +3,49 @@ id: deploying-a-configuration
...
 
-## Explanations
+4.  Check the following boxes (see section [**Export options**](#options-dexport)) :
 
-Multiple options are available in the configuration generation page:
+    - **Generate Configuration Files**
+    - **Run monitoring engine debug (-v)**
+    - **Move Export Files**
+    - **Restart Monitoring Engine**. Use the most appropriate method: 
+      - **Reload** : when you have created, deleted or edited supervised objects
```
Comment (tuntoja) [inline en/monitoring/monitoring-servers/deploying-a-configuration.md:27]:
```suggestion
      - **Reload** : when you have created, deleted or edited monitored objects
```

File: en/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 2]
```diff
@@ -3,39 +3,49 @@ id: deploying-a-configuration
...
+      - **Restart** : when you have made changes to the way a poller and the central server communicate, or
```
Comment (tuntoja) [inline en/monitoring/monitoring-servers/deploying-a-configuration.md:28]: the way a poller and the central server communicate together ?

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

File: en/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 3]
```diff
@@ -3,39 +3,49 @@ id: deploying-a-configuration
 title: Deploying a configuration
 ---
 
-On creation/deletion/change of objects via the configuration interface, the
-changes performed are not applied automatically to the scheduler. To apply the
-changes performed, it is necessary to follow the procedure below.
+When you create, delete or edit objects in the **Configuration** menu, changes are not applied automatically (neither on the central server on which you have made the change, nor on any remote server or poller linked to it). For the changes to be taken into account, you have to export the configuration.
 
-## Procedure
+For changes to remote servers or pollers, you must export the configuration on the central server.
```
Comment (tuntoja) [inline en/monitoring/monitoring-servers/deploying-a-configuration.md:8]:
```suggestion
For changes to remote servers or pollers, you must export their configuration on the central server.
```

File: fr/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 4]
```diff
@@ -3,44 +3,48 @@ id: deploying-a-configuration
...
-configuration, les changements effectués ne sont pas appliqués de manière
-automatique aux serveurs de supervision. Afin de pouvoir appliquer les
-modifications effectuées, il est nécessaire de suivre la procédure suivante
-ci-dessous.
+Lorsque vous créez, supprimez ou modifiez des objets via le menu
+**Configuration**, les changements effectués ne sont pas appliqués de manière
+automatique (ni sur le serveur central sur lequel vous avez créé, supprimé ou modifié l'objet, ni sur aucun collecteur ou serveur distant qui en dépendrait). Pour que les modifications soient prises en compte, il est nécessaire d'exporter la configuration.
 
-## Procédure
+Pour des changements apportés à des serveurs distants ou des collecteurs, l'export doit se faire sur le serveur central.
```
Comment (tuntoja) [inline fr/monitoring/monitoring-servers/deploying-a-configuration.md:10]:
```suggestion
Pour des changements apportés à des serveurs distants ou des collecteurs, l'export de leur configuration doit se faire sur le serveur central.
```

**Review by tuntoja — APPROVED** [review]

File: fr/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 5]
```diff
@@ -3,44 +3,48 @@ id: deploying-a-configuration
...
 
   - **Générer les fichiers de configuration** : Génère les fichiers de
-    configuration de l’ordonnanceur dans un répertoire temporaire. Cette
-    configuration est générée à partir des objets configurés via l’interface web
-  - **Lancer le débogage du moteur de supervision (-v)** : Permet à
-    l’ordonnanceur de vérifier la configuration générée
+    configuration du moteur de supervision dans un répertoire temporaire. Cette
+    configuration est générée à partir des objets configurés via l’interface web.
+  - **Lancer le débogage du moteur de supervision (-v)** : Permet au
+    moteur de supervision de vérifier la configuration générée.
```
Comment (omercier) [inline fr/monitoring/monitoring-servers/deploying-a-configuration.md:44]:
```suggestion
  - **Lancer le débogage du moteur de supervision (-v)** : Effectue une vérification de l'intégrité des fichiers de configuration du moteur de supervision.
```

File: fr/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 6]
```diff
@@ -3,44 +3,48 @@ id: deploying-a-configuration
...
   - **Deplacer les fichiers générés** : Déplace les fichiers de configuration du
```
Comment (omercier) [inline fr/monitoring/monitoring-servers/deploying-a-configuration.md:45]:
```suggestion
  - **Déplacer les fichiers générés** : Déplace les fichiers générés d'un répertoire temporaire vers le répertoire de configuration du moteur de supervision
```

File: fr/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 7]
```diff
@@ -3,44 +3,48 @@ id: deploying-a-configuration
...
-    répertoire temporaire vers le répertoire de l’ordonnanceur
-  - **Redémarrer l'ordonnanceur** : Redémarre l’ordonnanceur afin d’appliquer
-    les nouveaux fichiers de configuration
+    répertoire temporaire vers le répertoire du moteur de supervision.
```
Comment (omercier) [inline fr/monitoring/monitoring-servers/deploying-a-configuration.md:46]:
```suggestion
```

File: fr/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 8]
```diff
@@ -3,44 +3,48 @@ id: deploying-a-configuration
...
+  - **Redémarrer l'ordonnanceur** : Redémarre le moteur de supervision afin d’appliquer
```
Comment (omercier) [inline fr/monitoring/monitoring-servers/deploying-a-configuration.md:47]:
```suggestion
  - **Redémarrer l'ordonnanceur** : Redémarre ou recharge le moteur de supervision afin d’appliquer
```

File: fr/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 9]
```diff
@@ -3,44 +3,48 @@ id: deploying-a-configuration
...
+    les nouveaux fichiers de configuration.
```
Comment (omercier) [inline fr/monitoring/monitoring-servers/deploying-a-configuration.md:48]:
```suggestion
    la nouvelle configuration.
```

File: fr/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 10]
```diff
@@ -3,44 +3,48 @@ id: deploying-a-configuration
...
   - **Commande exécutée post-génération** : Exécute la commande post-génération
-    paramétrée au niveau de la configuration de l’ordonnanceur
+    paramétrée au niveau de la configuration du moteur de supervision.
```
Comment (omercier) [inline fr/monitoring/monitoring-servers/deploying-a-configuration.md:50]:
```suggestion
    paramétrée au niveau de la configuration du collecteur.
```

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

File: en/monitoring/monitoring-servers/deploying-a-configuration.md [hunk 11]
```diff
@@ -3,39 +3,49 @@ id: deploying-a-configuration
 title: Deploying a configuration
 ---
 
-On creation/deletion/change of objects via the configuration interface, the
-changes performed are not applied automatically to the scheduler. To apply the
-changes performed, it is necessary to follow the procedure below.
+When you create, delete or edit objects in the **Configuration** menu, changes are not applied automatically (neither on the central server on which you have made the change, nor on any remote server or poller linked to it). For the changes to be taken into account, you have to export the configuration.
 
-## Procedure
+For changes to remote servers or pollers, you must export their configuration on the central server.
```
[the remark this answers is not in the document]
Comment (tuntoja) [reply en/monitoring/monitoring-servers/deploying-a-configuration.md:8]: it means that for any changes made to pollers that are not the central itself, their configuration should be deployed from the central server's UI, and not from the Remote server's UI. Do you confirm this @omercier (background: lots of confusion during 21.04 campaign)

[the remark this answers is not in the document]
Comment (omercier) [reply en/monitoring/monitoring-servers/deploying-a-configuration.md:8]: I confirm, so i suggest putting it this way:

> Any configuration change on a host, a service or a poller must be performed and exported from the central server's web user interface or API.

VF:

> Tout changement de configuration sur un hôte, un service ou un collecteur devra se faire et s'exporter via l'interface utilisateur ou l'API du serveur central.

Would it not be better this way?

**Review by omercier — APPROVED** [review]

**Pull request merged**